<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>design</class>
 <widget class="QMainWindow" name="design">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>design</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextToTrain">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>490</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddConv">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Convolution</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddPooling">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Pooling</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddFully">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Dense</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>490</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>490</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>490</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>export</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>591</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="inputShaplabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>input shape：</string>
    </property>
   </widget>
   <widget class="QLabel" name="xlabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xlineEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ylineEdit_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ylabel_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zlineEdit_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="zlabel_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>40</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flattenButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Flatten</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>863</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>openChoiseLayer()</slot>
 </slots>
</ui>
